### DISCUSSION TITLE
Use to Chinese;This option is not available

### DISCUSSION THREAD

**Discussion by kamaitaching** [eclipse-theia/theia#10477] [General]
![image](https://user-images.githubusercontent.com/88232327/143612510-a6aa1f13-24f0-4667-bc4d-4877c33ab335.png)
This option is not available

**Comment by msujew** [comment]
It looks like you're running an older version of Theia (Based on the icons - most of them have been updated with version 1.18 of Theia). Localization is only fully supported after Theia 1.19.
